device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/InternalClient.java

```diff
@@ -26,17 +31,17 @@
 public class InternalClient
```

Comment:
Considering that we're starting to expose more public methods from this class do we want to change the visisbility so there is no confusion about the clients? ie, we only expose a DeviceClient, ServiceClient and ModuleClient?

Reply:
I'll mark this class and other similar classes package private once this PR is in, and once I get #1308 from master as well